tests/test.rs

```diff
@@ -5,6 +5,7 @@ fn no_items() {
     let grid = Grid::new(GridOptions {
         direction: Direction::TopToBottom,
         filling: Filling::Spaces(2),
+        tab_size: 8,
```

Comment:
why does it has to be mandatory ? :)
thanks

Reply:
Because it is stated in the [documentation](https://www.gnu.org/software/coreutils/manual/html_node/General-output-formatting.html) under the category '--tabsize=cols'.

In contrast, the statement in the coreutils "comment" claims that

`"Assuming cursor is at position FROM, indent up to position TO. Use a TAB character instead of two or more spaces whenever possible."`

So, there are two times (probably more) where tabs will be used instead of normal spaces:
- One is that when using `ls`, it will replace 8 spaces with 1 tab.
- Another is that when using `ls -CF`, it will replace 2 spaces with 1 tab

Reply:
sorry, I meant in the API :)

Reply:
Oh, I think I misread your question earlier. If I understand correctly, you're asking to keep the original API without modifying it. I think it can be achieved by introducing a builder pattern or some sort of struct wrapper. This way, the original API can be maintained while giving users the option to modify the tab size using a stuct wrapper or builder pattern, not directly modify the original API.

Apologize :)